jellyfin_kodi/dialogs/context.py

```diff
@@ -64,7 +64,7 @@ def onAction(self, action):
 
             if self.getFocusId() == LIST:
                 option = self.list_.getSelectedItem()
-                self.selected_option = option.getLabel()
+                self.selected_option = option.getLabel().decode('utf-8')
```

Comment:
We should be using `six.ensure_text()` in stead, this will fail if `.getLabel()` ever returns an `unicode` object.
We should also add `version="1.13.0"` to the six dependency, as this function got added somewhere between `v1.11 - 1.13`.